### PR TITLE
[Issue Tracker] Release fixes

### LINKED ISSUE #8552: Issue Tracker
**Describe the bug**
Issues can be created with all of the required fields working as expected. However, when you get an issue assigned, the user(neither the assignee nor the reporter) is unable to look into its contents. Trying to open the issue opens up a blank page. Linking a screenshot for reference

**To Reproduce**
Steps to reproduce the behavior (attach screenshots if applicable):
1. Go to Tools
2. Click on 'Issue Tracker'
3. Scroll down to any issue id. I have created ticket # 28 and 29
4. Try to open them by clicking the issue # or title

**What did you expect to happen?**
Ideally, you would expect to see the issue description with other relevant details such as PSCID, Visit Label, Category, Module, Attachments etc

**Browser Environment (please complete the following information):**
- OS: Mac)S Monterey version 12.6.3
- Browser : Chrome 

**Server Environment (if known):**
_Note: We only support the most recent release of LORIS._
- LORIS Version: [e.g. 22.0.0]
- Linux distribution and Version: [e.g. Ubuntu 16.04, CentOS 7]
- MySQL/MariaDB Version: [e.g. MySQL 5.7, MariaDB 10.3]

**Additional context**

### PR DESCRIPTION
**Pull request by laemtl**
Fixes  #8552
Also fixes some JS warnings, and another bug found along the way (Creating an issue without an assignee will successfully create an issue that won't display in the list of issues).

### CODE REVIEW

**Comment by xlecours** [comment]
@regisoc You can review after Laetitia fixes the Circle Ci errors.

**Comment by laemtl** [comment]
@regisoc Tests fixed.

File: modules/issue_tracker/jsx/IssueForm.js [hunk 1]
```diff
@@ -243,7 +243,10 @@ class IssueForm extends Component {
             label='Assignee'
             emptyOption={true}
             options={this.state.Data.assignees}
-            disabledOptions={this.state.Data.inactiveUsers}
+            disabledOptions={this.state.Data.inactiveUsers.reduce(
+              (disabledOpts, inactiveUser) =>
+                (disabledOpts[inactiveUser] = '', disabledOpts), {}
+            )}
```
Comment (regisoc) [inline modules/issue_tracker/jsx/IssueForm.js:249]: @laemtl I am having this error in js `Uncaught TypeError: this.state.Data.inactiveUsers.reduce is not a function`.

**Review by regisoc — APPROVED** [review]
LGTM

**Comment by regisoc** [comment]
LGTM again. Do the `.map` and `licence` files needed?

File: jsx/Form.js [hunk 2]
```diff
@@ -2221,9 +2221,9 @@ class StaticElement extends Component {
       <div className="row form-group">
         {label}
         <div className={this.props.class}>
-          <p className={this.props.textClass}>
+          <div className={this.props.textClass}>
```
Comment (driusan) [inline jsx/Form.js:2224]: Why is this changed? I think `<p>` is probably more accurate (it seems to be a paragraph in this context..) and changing Form.js affects every place that uses it.

**Review by driusan — CHANGES_REQUESTED** [review]
There also seem to be a punch of files accidentally commited (.js.map, .js.LICENSE.txt..)

File: jsx/Form.js (hunk 2 above)
Comment (laemtl) [reply jsx/Form.js:2224]: The text can contain another p which leads to an invalid nesting of p.

**Pull request merged**